src/services/cache/local-cache.ts

```diff
@@ -63,5 +63,6 @@ export const LocalCacheService = (): ICacheService => {
   return wrapAsyncFunctionsToRunInSpan({
     namespace: "services.cache.local",
     fns: { set, get, getOrSet, clear },
+    redactFnArgs: true,
```

Comment:
this is not good... why redact all feature that use cache?

Reply:
Using our previous method of redaction, we would have to redact `code.function.param.key` as well as `code.function.param.value` for all features that use the cache anyway.

Reply:
In addition to changing the api of the cache service to change single parameter functions into object parameters. For example changing `get(key)` to `get({key})`.

Reply:
cache interface probably was implemented previous the the params rule so make sense change the interface to use named params. Also since this is for the temp hack is easier add/remove redacted to config files